src/status_im/default_fleet.cljs

```diff
@@ -0,0 +1,6 @@
+(ns status-im.default-fleet
+  (:require-macros [status-im.utils.slurp :refer [slurp]]))
+
+(def default-fleets
+  (do (println "HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh")
```

Comment:
`HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH` is that a shadow-cljs magic incantation?

Reply:
was to check when module is loaded